### ISSUE TITLE
Watch for specific face?

### ISSUE THREAD

**Issue by hermano360** [tehnokv/picojs#9]
Is the program able to have data about faces and let you know about the confidence level of a matching face?

**Comment by berak** [comment]
@hermano360  no. it's purpose is to *detect* faces. what you want is called *face recognition*
